MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+recursive-include openfisca_web_api/i18n *.mo
+include *.md
+include LICENSE
+include *.ini
```

Comment:
Great, thanks a lot for the answer! I'm happy to see that you had a working solution for your problem  🙂  We'll document it so that others can learn, your review on the documentation will be welcome 😃